packages/ceramic-cli/src/ceramic-daemon.ts

```diff
@@ -89,6 +84,26 @@ class CeramicDaemon {
     return new CeramicDaemon(ceramic, opts)
   }
 
+  registerAPIPaths (app: any, gateway: boolean): void {
+    app.get(toApiPath('/versions/ceramic/:cid'), this.versions.bind(this))
+    app.get(toApiPath('/show/ceramic/:cid'), this.show.bind(this))
+    app.get(toApiPath('/state/ceramic/:cid'), this.state.bind(this))
+    app.get(toApiPath('/pin/ls/ceramic/:cid'), this.listPinned.bind(this))
+    app.get(toApiPath('/pin/ls'), this.listPinned.bind(this))
+
+    if (!gateway) {
+      app.post(toApiPath('/create'), this.createDocFromGenesis.bind(this))
+      app.post(toApiPath('/apply'), this.applyRecord.bind(this))
+      app.get(toApiPath('/pin/add/ceramic/:cid'), this.pinDocument.bind(this))
+      app.get(toApiPath('/pin/rm/ceramic/:cid'), this.unpinDocument.bind(this))
+    } else {
+      app.post(toApiPath('/create'), this.notSupported.bind(this))
```

Comment:
`/create` needs to be supported in gateway mode. Otherwise we won't be able to lookup account links.

Reply:
ok yeah, but that does dag.put, seems like another read only function needed after? but can just move for now

Reply:
on second glance, its not actually pinning, so this will be good for now